Tasks/MavenV2/maventask.ts

```diff
@@ -88,6 +88,7 @@ if (isWindows &&
 }
 
 tl.debug('Maven executable: ' + mvnExec);
+tl.checkPath(mvnExec, 'maven path');
```

Comment:
Should the checkPath be before the debug?

Reply:
I wanted to see what path we were checking for, so put the check after the debug.